experimental/dds/tree2/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -66,3 +72,32 @@ export function keyIsValidIndex(key: string | number, length: number): boolean {
 	if (typeof key === "string" && String(index) !== key) return false;
 	return Number.isInteger(index) && 0 <= index && index < length;
 }
+
+/**
+ * Retrieve a {@link StableNodeKey} on the given {@link EditableTree} node.
+ * @param nodeKeyFieldKey - the key of the field under which the node key resides
+ * @param node - the {@link EditableTree} node from which to retrieve the node key
+ * @returns the {@link StableNodeKey} on `node`, or undefined if there is none.
+ */
+export function getStableNodeKey(
```

Comment:
I'm not sure where this is used, but why doesn't this just take the local node key and convert it? It would likely have to have a context to get the compressor, but it will anyway since we won't be storing the string in-memory in the future.

Reply:
I used to use this in two places but now it's just a helper for editable tree, so I'll reduce the surface, thanks.

I'm not sure what you are asking. This function is currently _how_ we get the local node key. We first use this to get the stable key, and then convert to a local key. There is no local key stored anywhere right now.